From e8caca4ddfc7434c6f04432ba195781874a3c311 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Thu, 14 Jan 2021 14:47:53 +0800
Subject: [PATCH] Revert "arm64: rockchip_defconfig: enable HZ_1000"

This reverts commit 9a883c648aafcab3f03319e37f1659157ae7ffc7.

There is no need to keep HZ=1000 after commit 3be6453a90e4
("kthread: do not preempt current task if it is going to call schedule()").

Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: Id035e01cbf274dce0e6f12caef7f4eeb7efc2406
---
 arch/arm64/configs/rockchip_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index e02a97a13389..9dd5ba1e0b2b 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -65,7 +65,7 @@ CONFIG_PCIE_DW_ROCKCHIP=y
 # CONFIG_QCOM_FALKOR_ERRATUM_E1041 is not set
 CONFIG_SCHED_MC=y
 CONFIG_NR_CPUS=8
-CONFIG_HZ_1000=y
+CONFIG_HZ_300=y
 CONFIG_SECCOMP=y
 CONFIG_ARMV8_DEPRECATED=y
 CONFIG_SWP_EMULATION=y
-- 
2.35.3

